src/main/RP.ts

```diff
@@ -54,12 +51,12 @@ export class RP {
       claims?: ClaimOpts;
     }
   ): Promise<VerifiedAuthenticationResponseWithJWT> {
-    return AuthenticationResponse.verifyJWT(jwt, this.newVerifyAuthenticationResponseOpts(opts));
+    return AuthenticationResponse.verifyJWT(jwt, this.newVerifyAuthenticationResponseOpts(opts), this._authRequestOpts.linkedDomainValidationMode);
```

Comment:
This should not become a third param. Should be handled by the newVerifyAuthResponseOpts method